ui/app/controllers/vault.js

```diff
@@ -6,12 +6,6 @@
 import Controller from '@ember/controller';
 
 export default class VaultController extends Controller {
-  queryParams = [
-    {
-      wrappedToken: 'wrapped_token',
```

Comment:
I'm not really sure why this was here...the `wrapped_token` param was only used by auth-form which is only accessible at `vault/auth`. Moved it to the pertinent route/controller `vault.auth`